internal/xds/translator/testdata/out/xds-ir/authn.routes.yaml

```diff
@@ -0,0 +1,31 @@
+- name: first-listener
```

Comment:
Per https://github.com/envoyproxy/gateway/pull/813#discussion_r1053876076, this PR uses per filter config to configure the JWT provider for matched traffic. @lizan @arkodg PTAL when you have a moment, PTAL and let me know if this looks correct to you. I did not include any `match` in `requirements` since the route provides the match.

Reply:
yeah look good for a config perspective